content/ja/docs/collector/distributions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ディストリビューション
+weight: 25
+default_lang_commit: 8d115a9df96c52dbbb3f96c05a843390d90a9800
+---
+
+OpenTelemetryプロジェクトは現在、コレクターの[事前ビルド済みディストリビューション][pre-built distributions]を提供しています。
+各[ディストリビューション][distributions]に含まれるコンポーネントは、それぞれのディストリビューションの`manifest.yaml`で確認できます。
+
+[pre-built distributions]: https://github.com/open-telemetry/opentelemetry-collector-releases/releases
+[distributions]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions
+
+{{% docs/latest-release collector-releases /%}}
```

Comment:
FYI, I've created the following issue to handle the localization of the named shortcode:

- #6460

But this PR is ok as is.